electron/helpers/utils.js

```diff
@@ -10,8 +11,45 @@ export {
   v4 as generateKey
 } from 'uuid'
 
+const harmfulSwitches = [
+  'remote-debugging-port',
+  'inspect',
+  'inspect-brk',
+  'inspect-brk-node',
+  'inspect-port',
+  'inspect-publish-uid',
+  'js-flags',
+  'proxy-server',
+  'host-rules'
+]
+
+function checkHarmfulSwitchPresence () {
```

Comment:
@xyloflake IMO, functions returning boolean better be named like `isAnyHarmfulSwitches`.

Reply:
Sure! Updating in next commit!

Reply:
I've decided to call it `areHarmfulSwitchesPresent`

Reply:
@xyloflake Please follow this repo's convention.
I use `is` event if it's plural. (For unification).

Reply:
Sure. Will be fixed in next commit.